### ISSUE TITLE
Crashes on any JZ4760B devices

### ISSUE THREAD

**Issue by gameblabla** [Nebuleon/ReGBA#16]
For reasons not clear, ReGBA crashes as soon as it tries to boot a game on any LDK/RS-97/RG-300 using an Ingenic JZ4760B.
GPSP (pcerceui's version and forks based on it) still manage to boot but even the other gpsp versions crash in the same fashion.

It doesn't matter if MIPS_XBURST is defined or not, and it crashes at the recompiler, making it difficult to debug with GDB.

**Comment by Nebuleon** [comment]
Did you try to compile without `MIPS_32R2`? It's a compile-time variable used in `source/mips/stub.S` and `source/mips/emit.h`.

I just went and looked, and either the JZ4760B is a MIPS32 processor that declares conformance to Release 2 instructions but fails to execute them, or is a MIPS32 (Release 1) processor outright.

(I also don't have any device here that uses the JZ4760B, so it's difficult for me to test. I'd welcome a patch that adds `source/opendingux/Makefile.ldk` without `-DMIPS_32R2` if that works for you.)

**Comment by gameblabla** [comment]
Yes, i tried it against the very last commit and not enabling MIPS_32R2 however it still crashes shortly after loading the game in memory. It doesn't matter if you use GCC 4.7 or GCC 9.2.
The JZ4760B is full of bugs, including one you just pointed out or another one where the CPUID says there's no FPU even though it does have one...

I'm running out of ideas to get it running : (
I mean what does this 
https://github.com/gameblabla/gpsp/blob/master/mips_stub.S

Does different that could impact running it compared to other MIPS GPSP forks i have no idea.

**Comment by pcercuei** [comment]
Did you try the Dingoo A320 build?

**Comment by gameblabla** [comment]
I'm not sure what you mean by that, it's the same thing ? It's not like there has been a binary release of it for the Dingoo. And yes, i did in fact use the Makefile.dingoo and it still crashes.
